integritybackend/config.py

```diff
@@ -11,8 +11,13 @@
 # Secret for encoding/decoding JWT tokens.
 JWT_SECRET = os.environ.get("JWT_SECRET")
 
-# Full path to claim_tool binary. Must be already configured.
-CLAIM_TOOL_PATH = os.environ.get("CLAIM_TOOL_PATH")
+# Full path to c2patool binary. Must be already configured.
+C2PATOOL_PATH = os.environ.get("C2PATOOL_PATH")
+# C2PA cert and key files
+C2PA_PRIVATE_KEY_PATH = os.environ.get("C2PA_PRIVATE_KEY_PATH")
+C2PA_CERT_CHAIN_PATH = os.environ.get("C2PA_CERT_CHAIN_PATH")
```

Comment:
I think currently we have the `starling-lab_c2pa-key` key under `c2pa-proofmode` and `c2pa-starling-capture` that defines the path to the key. This allows different collections to be signed with different  keys.

I'm not if defining it to 1 signature  per system changes any requirements.

Reply:
In the org/collection config we do point to a particular authsigner, for example. We can do this the same with c2pa signers.

From sec perspective, it's not good to change this, as it should represent the org (us) and consistent across permissions, but I think in reality we may have different keys. Given how expensive it is to buy a cert, I think we'll likely stay with one cert. This feature to have variable key is probably valuable only for QA and not for prod.

Reply:
The cert and the key must change together. Maybe the variable names are not clear sorry, I am trying to keep them similar to what c2patool uses.

`C2PA_CERT_CHAIN_PATH` is the certificate that holds the public key. This is akin to a domain certificate.

`C2PA_PRIVATE_KEY_PATH` is another PEM file, and it holds the private key for the certificate.

It's similar to how with the old claim_tool, you'd have two files as well that you generated with openssl. They needed to stay together. 	`openssl req -new -newkey rsa:4096 -days 180 -nodes -x509 -keyout ~/.cai/temp_key.pem -out ~/.cai/temp_key.pub`

Reply:
This is fixed in an update, to allow more than one cert/key pair. See below.